web/sass/custom.scss

```diff
@@ -1,85 +1,120 @@
-$light-blue: #2d83ba;
-$orange: #F09740;
-$dark-blue: #053E4C;
-$grey: #242A2F;
-$text: #FFF;
-
-body {
-    color: $text;
-    font-family: "Lato", sans-serif;
-    font-size: 16px;
+//
+// Variables
+// --------------------------------------------------
+$light-blue: #2d83ba;  // $body-bg
+$dark-blue:  #052E4C;  // $brand-primary
+$orange:     #F09740;  // $brand-success
+$gray:       #242A2F;  // $gray
```

Comment:
not with the way we were using compass and linking the stylesheets. compass was watching all *.scss files, excluding those prefixed with an underscore, and compiling them to their own css. So styles.scss -> styles.css and custom.scss -> custom.css completely separate from each other, while _bootstrap-variables.scss was simply included in styles.scss. I rearranged things so custom.scss is now also an import in styles.scss, which means we only have to link the one and only compiled css file. i'll update the PR